File: dotnet/BlazorApp1/Data/Pokemon.cs
namespace BlazorApp1.Data;

public class Pokemon
{
    public Guid Id { get; set; }
    public string Name { get; set; } = "";
    public int DexId { get; set; }
    public int Upvotes { get; set; }
    public int Downvotes { get; set; }
}

File: fsharp-fun-blazor/src/Db/Pokemon.fs
namespace FSharpFunBlazor.Db

open System

[<CLIMutable>]
type Pokemon = {
    Id: int
    Name: string
    UpVotes: int
    DownVotes: int
    DexId: int
    InsertedAt: DateTime
    UpdatedAt: DateTime
}
